File: src/EducationSystem/EducationSystem.Models/Source/Rest/ProgramData.cs
using EducationSystem.Models.Source.Rest.Base;

namespace EducationSystem.Models.Source.Rest
{
    public class ProgramData : Model
    {
        public int ProgramId { get; set; }

        public string Input { get; set; }

        public string ExpectedOutput { get; set; }
    }
}

File: src/EducationSystem/EducationSystem.WebApp/App/src/models/ProgramData.ts
import Model from './Model'
import Program from './Program'

export default class ProgramData extends Model {
  public ProgramId?: number
  public Program?: Program
  public Input: string = ''
  public ExpectedOutput: string = ''
  
  constructor(input: string, expectedOutput: string) {
    super()
    
    this.Input = input
    this.ExpectedOutput = expectedOutput
  }
}